pkg/server/handler/tikvhandler/tikv_handler.go

```diff
@@ -2166,3 +2168,47 @@ func (h IngestConcurrencyHandler) ServeHTTP(w http.ResponseWriter, req *http.Req
 		handler.WriteError(w, errors.New("method not allowed"))
 	}
 }
+
+// GCHandler is the handler for GC related API.
+type GCHandler struct {
+	store kv.Storage
+}
+
+// NewGCHandler creates a GCHandler.
+func NewGCHandler(store kv.Storage) *GCHandler {
+	return &GCHandler{
+		store: store,
+	}
+}
+
+// ServeHTTP implements the HTTP handler interface.
+func (gc *GCHandler) ServeHTTP(w http.ResponseWriter, _ *http.Request) {
```

Comment:
Loading GC states should only work for `GET` requests. Consider rejecting other HTTP methods by 405 Method Not Allowed.